docs/examples/03_building_custom_image.ipynb

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
nit: the rest of the procedural notebooks are in `docs/getting_started/experimental`, whereas the other ones here (VQE / QAOA) are more application-like ... which begs the question of whether we need to think about how to organize our documentation (i.e. should we start making sections for end users, for function providers, etc.)

Reply:
And the pending move to the qiskit org probably provides a good cover for a docs reorg ...

Reply:
Agree with you :) 
I also was trying to figure out where to put those. I was trying to decide between `examples` or `deployment`

Reply:
Finally moved it to deployment as rst document.